docs/download/76.md

```diff
@@ -242,17 +242,16 @@ ICU4J should work on Android API level 21 and later but may require “[library
 
 ## Download
 
-Source and binary downloads are available on the git/GitHub tag page: https://github.com/unicode-org/icu/releases/tag/release-76-rc
+Source and binary downloads are available on the git/GitHub tag page: <https://github.com/unicode-org/icu/releases/tag/release-76-rc>
 
 See the [Source Code Setup](../devsetup/source/) page for how to download the ICU file tree directly from GitHub.
 
 ICU locale data was generated from CLDR data equivalent to:
 
-* https://github.com/unicode-org/cldr/releases/tag/release-46-beta3
-* https://github.com/unicode-org/cldr-staging/releases/tag/release-46-beta3
+* <https://github.com/unicode-org/cldr/releases/tag/release-46-beta3>
```

Comment:
```suggestion
* <https://github.com/unicode-org/cldr/releases/tag/release-46>
```

Reply:
ok -- I got confirmation that for ICU beta3 is really what we have, but the anticipated final-release URL looks cleaner

done